src/player.js

```diff
@@ -1228,8 +1228,24 @@ export default class Player extends FakeEventTarget {
    * @private
    */
   _setDefaultTrack(type: string, language: string, defaultTrack: Track): void {
+    /**
+     * Comparing language string according to their length.
+     * @param {string} inputLang - The configured language.
+     * @param {string} trackLang - The default track language.
+     * @returns {boolean} - Whether the strings are equal or starts with the same substring.
+     */
+    function langComparer(inputLang: string, trackLang: string): boolean {
```

Comment:
if this is a utility for text track so move it to textTrack class as static mehod and use it:
```javascript
TextTrack.langComparer(language, track.language)
```